# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2025-02-20 22:19-0300\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

#: manage_translations.py:117
msgid "Error executing command"
msgstr "Erro ao executar o comando"

#: manage_translations.py:129
msgid "Choose a language"
msgstr "Escolha um idioma"

#: manage_translations.py:155
msgid "Adding language"
msgstr "Adicionando Idioma"

#: manage_translations.py:164
msgid "Language already exists"
msgstr "Idioma já existe"

#: manage_translations.py:175 manage_translations.py:207
msgid "No Python files found for extraction"
msgstr "Nenhum arquivo Python encontrado para extração"

#: manage_translations.py:192
msgid "Language added successfully!"
msgstr "Idioma adicionada com sucesso!"

#: manage_translations.py:196
msgid "Updating translations..."
msgstr "Atualizando traduções..."

#: manage_translations.py:201
msgid "Python files found:"
msgstr "Arquivos Python encontrados:"

#: manage_translations.py:212
msgid "Extracting strings to .pot file..."
msgstr "Extraindo strings para arquivo .pot..."

#: manage_translations.py:218
msgid "Updating existing translations..."
msgstr "Atualizando traduções existentes..."

#: manage_translations.py:225
msgid "File not found:"
msgstr "Arquivo nao encontrado:"

#: manage_translations.py:226
msgid "Creating a new"
msgstr "Criando um novo"

#: manage_translations.py:231
msgid "Compiling .mo file..."
msgstr "Copilando .mo file..."

#: manage_translations.py:248
msgid "Translations updated successfully!"
msgstr "Traduções atualizadas com sucesso!"

#: manage_translations.py:261
msgid "Invalid command"
msgstr "Comando inválido"

#: manage_translations.py:265
msgid "No arguments provided"
msgstr "Nenhum argumento fornecido"

#: task_commit/core.py:24
msgid "y"
msgstr "s"

#: task_commit/core.py:25
msgid "n"
msgstr "n"

#: task_commit/core.py:27
msgid "Starting commit process"
msgstr "Iniciando processo de commit"

#: task_commit/core.py:37
msgid "No changes to commit"
msgstr "Nenhuma mudança para commit"

#: task_commit/core.py:44
msgid "Do you want to add all changes"
msgstr "Deseja adicionar todas as mudanças"

#: task_commit/core.py:63
msgid "Manually add the changes and run the command again"
msgstr "Adicione manualmente as mudanças e execute o comando novamente"

#: task_commit/core.py:73 task_commit/core.py:208 task_commit/core.py:253
msgid "Invalid option"
msgstr "Opção inválida"

#: task_commit/core.py:80
msgid "New functionality"
msgstr "Nova funcionalidade"

#: task_commit/core.py:81
msgid "Bug fix"
msgstr "Correção de bug"

#: task_commit/core.py:82
msgid "Code refactoring"
msgstr "Refatoração de código"

#: task_commit/core.py:83
msgid "Documentation update"
msgstr "Atualização de documentação"

#: task_commit/core.py:84
msgid "Style changes"
msgstr "Alterações de estilo"

#: task_commit/core.py:85
msgid "Performance improvements"
msgstr "Melhorias de desempenho"

#: task_commit/core.py:86
msgid "Test addition/correction"
msgstr "Teste de adição/correção"

#: task_commit/core.py:87
msgid "Configuration changes"
msgstr "Alterações de configuração"

#: task_commit/core.py:88
msgid "Changes in continuous integration"
msgstr "Alterações na integração contínua"

#: task_commit/core.py:101
msgid "Choose commit type"
msgstr "Escolha o tipo de commit"

#: task_commit/core.py:120
msgid "Invalid commit type"
msgstr "Tipo de commit inválido"

#: task_commit/core.py:125
msgid "Process interrupted. Exiting..."
msgstr "Processo interrompido. Saindo..."

#: task_commit/core.py:130
msgid "Unexpected error occurred"
msgstr "Erro inesperado ocorrido"

#: task_commit/core.py:138
msgid "Which module was changed? (example: core, api, models): "
msgstr "Qual módulo foi alterado? (exemplo: core, api, models): "

#: task_commit/core.py:153
msgid "Module is mandatory"
msgstr "Módulo é obrigatório"

#: task_commit/core.py:161
msgid "Enter commit message"
msgstr "Digite a mensagem do commit"

#: task_commit/core.py:168
msgid "Commit message is mandatory"
msgstr "Mensagem de commit é obrigatória"

#: task_commit/core.py:179
msgid "Error: Git username not set"
msgstr "Erro: Nome de usuário do Git não configurado"

#: task_commit/core.py:188
msgid "Do you want to send the commit"
msgstr "Deseja fazer commit para o repositório"

#: task_commit/core.py:215
msgid "Commit canceled"
msgstr "Commit cancelado"

#: task_commit/core.py:219
msgid "Do you want to push to the repository"
msgstr "Deseja fazer push para o repositório"

#: task_commit/core.py:249
msgid "Push canceled"
msgstr "Push cancelado"

#: task_commit/core.py:260
msgid "Leaving..."
msgstr "Saindo..."

#: task_commit/core.py:265
msgid "Unexpected error"
msgstr "Erro inesperado"

#: task_commit/init.py:61
msgid "❌ .git/hooks directory not found. Please run inside a Git repository."
msgstr ""
"❌ Diretório .git/hooks não encontrado. Por favor, execute dentro de um "
"repositório Git."

#: task_commit/init.py:71
msgid "✅ Commit-msg hook successfully configured!"
msgstr "✅ Hook Commit-msg configurado com sucesso!"

#: task_commit/utils.py:77
msgid "Git user is required"
msgstr "Usuário do Git é obrigatório"

#: task_commit/utils.py:85
msgid "Error getting user from Git"
msgstr "Error getting user from Git"

#: task_commit/utils.py:105 task_commit/utils.py:162
msgid "Error checking Git status"
msgstr "Erro ao obter usuário do Git"

#: task_commit/utils.py:144
msgid "Changes not staged"
msgstr "Mudanças não encenadas"

#: task_commit/utils.py:149
msgid "Changes staged"
msgstr "Mudanças não encenadas"

#: task_commit/utils.py:154
msgid "Untracked files"
msgstr "Arquivos não rastreados"

#: task_commit/utils.py:179
msgid "Gitflow not installed, but push is successful"
msgstr "Gitflow nao instalado, mas o push é realizado"

#: task_commit/utils.py:202
msgid "Error getting current branch"
msgstr "Erro ao obter branch atual"

#: task_commit/utils.py:218
msgid "All changes added"
msgstr "Todas as mudanças adicionadas"

#: task_commit/utils.py:221
msgid "Error adding changes"
msgstr "Erro ao adicionar mudanças"

#: task_commit/utils.py:256
msgid "Commit successful"
msgstr "Commit realizado com sucesso"

#: task_commit/utils.py:259
msgid "Error committing"
msgstr "Erro ao realizar commit"

#: task_commit/utils.py:283
msgid "Do you want to 'publish' or 'finish' this branch?"
msgstr "Deseja 'publish' ou 'finish' essa branch"

#: task_commit/utils.py:301
msgid "Error publishing branch"
msgstr "Erro ao publicar branch"

#: task_commit/utils.py:309
msgid "Error finalizing branch"
msgstr "Erro ao finalizar branch"

#: task_commit/utils.py:312
msgid "Invalid action"
msgstr "Ação inválida"

#: task_commit/utils.py:332
msgid "Error when pushing"
msgstr "Erro ao fazer push"

#~ msgid "CI changes"
#~ msgstr "Alterações na integração contínua"
